### ISSUE TITLE
About dialog missing icon in the 0.11 Debian package

### ISSUE THREAD

**Issue by mgedmin** [gtimelog/gtimelog#110]
I get this on startup:

```
(gtimelog:18979): Gtk-WARNING **: Could not load image 'gtimelog.png': Nepavyko atverti failo „/usr/share/gtimelog/gtimelog.png“: No such file or directory
```

and the About dialog is missing the package.

The problem doesn't occur on git master.

**Comment by mgedmin** [comment]
A fix is to edit `/usr/share/gtimelog/*.ui` and change `gtimelog.png` to `/usr/share/pixmaps/gtimelog.png` in there.

**Comment by mgedmin** [comment]
Should be fixed in 0.11-4mg2 that I've just uploaded to the PPA.
